### DISCUSSION TITLE
The switch displays the focus color after clicking.

### DISCUSSION THREAD

**Discussion by kai-orion** [material-components/material-web#5508] [General]
In desktop browsers, why does switch appear focused after being clicked? When I was browsing the [switch accessibility](https://m3.material.io/components/switch/accessibility) document, I noticed that the switch in the document was in focus after being clicked. There is no focus state, and in the actual code, the background color of the handle will maintain the color of the focus state after the switch is clicked.

This is a screenshot after clicking and removing focus:
![image](https://github.com/material-components/material-web/assets/67639369/c4dc71ae-71b4-4de3-a56b-761219b17d59)

Here is a screenshot of what does nothing when clicked:
![image](https://github.com/material-components/material-web/assets/67639369/1a45380c-efd1-4976-bca8-5b1d7b822e07)

**Comment by asyncliz** [comment]
On the web, [`:focus` is generally applied when a user clicks or taps](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus) on an element. `:focus` persists until another element is clicked on to move the focus.

There is one notable exception. Safari intentionally does not apply `:focus` styles after a pointer click as a design opinion. This differs from Chrome, Edge, and Firefox.

If you're theming and noticing the colors changing on hover/focus/press when you set the color, check out ["How do I change the color of a button?"](https://github.com/material-components/material-web/blob/main/docs/support.md#how-do-i-change-the-color-of-a-button) and ["Why does my color change on hover/focus/pressed?"](https://github.com/material-components/material-web/blob/main/docs/support.md#why-does-my-color-change-on-hoverfocuspressed)
